src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/Thread.java

```diff
@@ -89,6 +89,7 @@ public long allocatedBytes() {
   public boolean   isServiceThread()             { return false; }
   public boolean   isMonitorDeflationThread()    { return false; }
   public boolean   isAttachListenerThread()      { return false; }
+  public boolean   isDeoptimizeObjectsALotThread() { return false; }
```

Comment:
Is this method used anywhere?

Reply:
No, but I added it to be consistent with all the other JavaThread subclasses. Actually none of these are used except for isCompilerThread and isCodeCacheSweeperThread. The latter reference is irrelevant since there is no longer a CodeCacheSweeperThread, and the former reference should be replaced with isHiddenFromExternalView(). These are all things documented to fix as part of [JDK-8348347](https://bugs.openjdk.org/browse/JDK-8348347), but in the meantime I though it best to be consistent with existing code.